### PR TITLE
Add ListItemArticle

### PR DESCRIPTION
**Pull request by ludwig-pro**
## Checklist before requesting a review
- [x] Working on iOS
- [x] Working on Android
- [x] Integration tests added
- [x] Visual regressions screenshots are up to date
- [x] Design validated

### CODE REVIEW

**Comment by ludwig-pro** [comment]
question in progress : https://www.figma.com/file/tOJLNigXQXJqCSKgJFPZn5?node-id=4253:7007#442962482

**Comment by ianrobertlambert** [comment]
It looks great @ludwig-pro, thanks! I checked both iOS and Android.

### Feedback
- [x] Can we please change the chevron color to `Grey-400`?
- [x] Can we please use a `Primary` spinner on this ListItem? It should be the Bluko version.
- [x] Is it possible to use a `MD` spinner here (24x24)? I know it is larger than the chevron icon, but is there a way to do this by adjusting the padding maybe?

### Notes
- Based on your Slack message last week, I’ve created [additional states inside Figma](https://www.figma.com/file/tOJLNigXQXJqCSKgJFPZn5/App-Streamline-Lukomponents?type=design&node-id=4253%3A7007&t=Xr3dwwV3KU4DxDNc-1). Would it be possible to add the other state please? We now have:
  - [X] Default
  - [X] Touch
  - [x] Disabled
  - [X] Loading
  - [X] Skeleton
- I can create a separate ticket if this was not part of the original scope?

File: packages/streamline/src/components/list-items/list-item-article/list-item-article.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        description="Description"
+        onPress={onPress}
+      />
+    );
+
+    // When
+    fireEvent.press(getByText('Header'));
+
+    // Then
+    await waitFor(() => expect(onPress).toHaveBeenCalledTimes(1));
```
Comment (kimak) [inline packages/streamline/src/components/list-items/list-item-article/list-item-article.spec.tsx:31]: missing tests case:

- getByText('Title')
- getByText('Description')

And imageSource using accessibility text?

**Review by kimak — CHANGES_REQUESTED** [review]

**Comment by ludwig-pro** [comment]
> * is there a way to do this by adjusting the padding maybe?

I'm not sure to understand what you want 🤔 @ianrobertlambert

**Comment by ludwig-pro** [comment]
latest screenshot 

Simulator Screen Shot - iPhone 12 - 2023-05-15 at 15 03 09

**Comment by ianrobertlambert** [comment]
> > * is there a way to do this by adjusting the padding maybe?
> 
> I'm not sure to understand what you want 🤔 @ianrobertlambert

@ludwig-pro these look great, thanks a lot! ✨

If we look at this screenshot:

image

Sorry if I wasn't clear earlier. The `ChevronFarRight` icon is 16px x 16px, while the `MD Spinner` is 24px x 24px. How did you adjust the size difference between the two states? Did you decrease the space between the text and the area on the right side by -8px to accommodate the larger size of the spinner? I'm just trying to better understand how you built it. 😊

image

**Comment by ludwig-pro** [comment]
my global container have a padding, 

then I have a vertical stack with a <image> and a <text + icon> where <text + icon> have a padding left

and finally I have a vertical stack for the <text> and <the icon> when the text have a flex 1 ( so it's grow to fullfilled ) and the icon have a padding left

image

File: packages/streamline/src/components/list-items/list-item-article/components/image-spinner.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { Easing, FadeIn } from 'react-native-reanimated';
+
+import { AnimatedBox } from '../../../../primitives/animated-box/animated-box';
+import { Box } from '../../../../primitives/box/box';
+import Spinner from '../../../spinner/spinner';
+
+/**
+ * Todo - Use pressable from react-native-ama when issue below fixed
+ * https://github.com/FormidableLabs/react-native-ama/issues/92
```
Comment (hcourthias) [inline packages/streamline/src/components/list-items/list-item-article/components/image-spinner.tsx]: Unrelated comment ?

**Review by hcourthias — APPROVED** [review]

File: packages/streamline/src/components/list-items/list-item-article/components/image-spinner.tsx (hunk 2 above)
Comment (ludwig-pro) [reply packages/streamline/src/components/list-items/list-item-article/components/image-spinner.tsx]: 👀 thanks

**Review by kimak — APPROVED** [review]

**Pull request merged**